index.js

```diff
@@ -127,11 +156,20 @@ export function openDittoTimely(eventName, params = {}, env = "prod") {
 
   // set content
   modal.setContent(
-    `<iframe id="timely-iframe" style="width: 100%;height:100%;border:none;" src="${timelyUrlWithParams}"></iframe>`
+    `
```

Comment:
we can stop the execution of the function if eventName is missing, The error message can have the plugin name to indicate the message is from the plugin
something like : 'vanilla-timely: Event name is not provided.'